pkg/rpc/server.go

```diff
@@ -180,27 +181,25 @@ Methods:
 
 		results = peers
 
-	case "validateaddress", "getblocksysfee", "getcontractstate", "getrawmempool", "getrawtransaction", "getstorage", "submitblock", "gettxout", "invoke", "invokefunction", "invokescript", "sendrawtransaction", "getaccountstate":
+	case "validateaddress", "getblocksysfee", "getcontractstate", "getrawmempool", "getrawtransaction", "getstorage", "submitblock", "gettxout", "invoke", "invokefunction", "invokescript", "sendrawtransaction":
 		results = "TODO"
 
 	case "getassetstate":
 		var err error
-
-		param, exists := reqParams.ValueAt(0)
+		param, exists := reqParams.ValueAtAndType(0, "string")
 		if !exists {
-			err = errors.New("Param at index at 0 doesn't exist")
+			err = fmt.Errorf("Please provide a valid string assetID parameter")
```

Comment:
> Error strings should not start with a capital letter because they'll often be prefixed before printing:

https://github.com/golang/go/wiki/Errors

Reply:
and, just use https://github.com/pkg/errors

Reply:
okay, done.